packages/account-v2/src/components/base/WalletText/WalletText.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { CSSProperties, ElementType, ReactNode } from 'react';
+import classNames from 'classnames';
+import { TGenericSizes } from '../types';
+import './WalletText.scss';
+
+export interface WalletTextProps {
+    align?: CSSProperties['textAlign'];
+    as?: ElementType;
+    children: ReactNode;
+    color?: CSSProperties['color'] | 'error' | 'general' | 'less-prominent' | 'primary' | 'success' | 'warning';
+    fontStyle?: CSSProperties['fontStyle'];
+    lineHeight?: TGenericSizes;
+    size?: Exclude<TGenericSizes, '3xs' | '6xl' | '7xl'>;
+    weight?: CSSProperties['fontWeight'];
+}
+
+const WalletText: React.FC<WalletTextProps> = ({
+    align = 'left',
+    as = 'span',
+    children,
+    color = 'general',
+    fontStyle = 'normal',
+    lineHeight,
+    size = 'md',
+    weight = 'normal',
+}) => {
+    const textClassNames = classNames(
+        'wallet-text',
+        `wallets-text__size--${size}`,
+        `wallets-text__weight--${weight}`,
+        `wallets-text__align--${align}`,
+        `wallets-text__color--${color}`,
+        `wallets-text__line-height--${lineHeight}`,
+        `wallets-text__font-style--${fontStyle}`
+    );
+
+    const Tag = as;
+
+    return <Tag className={textClassNames}>{children}</Tag>;
```

Comment:
Text component is already available in deriv-com/ui library.

Reply:
I just copied the Wallet component as it it from the Wallet package since its not sharable. This is a temp stopgap until a Dropdown component is Available. If I'm coping the component as it is, do I need to make changes to it?